src/Data/Maybe.agda

```diff
@@ -56,6 +57,29 @@ monadPlus {f} = record
   nothing ∣ y = y
   just x  ∣ y = just x
 
+
+------------------------------------------------------------------------
+-- Other associated operations
+
+fromMaybe : {ℓ : Level} {A : Set ℓ} (default : A) (try : Maybe A) → A
+fromMaybe def (just x) = x
+fromMaybe def nothing  = def
+
```

Comment:
Note that this is in #359

Reply:
`fromMaybe` has now been merged in.

Reply:
Too fast... `fromMaybe` is really just a variable of `maybe`. Wouldn't it be better to direct people to that? At least implement it as
```agda
fromMaybe def v = maybe def id v
```
?

Reply:
In #359 we indeed had [`fromMaybe = maybe′ id`](https://agda.github.io/agda-stdlib/Data.Maybe.Base.html#1662)!

Reply:
indeed!